django/views/generic/base.py

```diff
@@ -62,9 +62,7 @@ def view(request, *args, **kwargs):
             self = cls(**initkwargs)
             if hasattr(self, 'get') and not hasattr(self, 'head'):
                 self.head = self.get
-            self.request = request
-            self.args = args
-            self.kwargs = kwargs
+            self.init(request, *args, **kwargs)
```

Comment:
Since we moved setting these attributes into `init()`, is it worth an `assert` afterwards to make sure I didn't override it and forget to call `super()`?

Reply:
Sure, it avoids inattention mistakes and diving in Django code to understand what's wrong. Without the assert, the error is very explicit (e.g. `AttributeError: 'LoginView' object has no attribute 'request'`), but an assert message helps understanding the cause.

If we add the assertion, do you think the note in the doc about calling `super()` is still helpful?

Reply:
Personally yes. (But maybe the examples in the docs will be enough)